int32 DIGIN 
float64 AIN0
float64 AIN1
